Logos/alcide_logo_06_080318_export_vertical.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 500 500"><defs><style>.a{fill:#0031e2;}</style></defs><title>alcide_logo_06_080318_export</title><path class="a" d="M224.43,203.2l25.7,25.58,25.7-25.58-25.7-25.58Zm149.8,18.92L250.36,92.52,126.5,222.12l9.41,8.93c2.07,2,51.69,48.46,114.45,48.46S362.74,233,364.82,231.05ZM250.36,253.4c-37.08,0-70-20.84-85.88-32.84l85.88-89.86,85.89,89.86C320.36,232.57,287.44,253.4,250.36,253.4ZM142.64,329.65v8.66a19.42,19.42,0,0,0-7.24-7.67,20,20,0,0,0-10.61-2.85A21,21,0,0,0,115,330.1a24.15,24.15,0,0,0-7.76,6.31,30,30,0,0,0-5.12,9.48,36.54,36.54,0,0,0-1.84,11.73,35.45,35.45,0,0,0,1.91,11.9A29.13,29.13,0,0,0,107.6,379a24.57,24.57,0,0,0,8.29,6.3,24.84,24.84,0,0,0,10.61,2.24,18,18,0,0,0,9.66-2.46,18.58,18.58,0,0,0,6.49-7.19v8.56h16.89V329.65Zm0,35.09a18.56,18.56,0,0,1-4.95,5.86,11,11,0,0,1-6.76,2.15,12.14,12.14,0,0,1-5.28-1.16,12.76,12.76,0,0,1-4.17-3.12,14.88,14.88,0,0,1-2.75-4.66,16.82,16.82,0,0,1-1-5.87,17.94,17.94,0,0,1,.9-5.7,15.52,15.52,0,0,1,2.58-4.88,13.21,13.21,0,0,1,4-3.4,10.14,10.14,0,0,1,5-1.26,11.49,11.49,0,0,1,7.38,2.74,16.5,16.5,0,0,1,5.07,7.35ZM227.2,371a10.74,10.74,0,0,1-5.74,1.61,10.52,10.52,0,0,1-4.74-1.11,11.89,11.89,0,0,1-3.92-3.09,14.55,14.55,0,0,1-2.61-4.8,19.73,19.73,0,0,1-.94-6.23q0-6.84,3.5-10.93a11,11,0,0,1,8.71-4.08,10.35,10.35,0,0,1,9.48,5.51l16.27-5.19a26.75,26.75,0,0,0-9.7-11.09q-6.47-4.25-16-4.24a30.48,30.48,0,0,0-12.46,2.43,27.2,27.2,0,0,0-9.23,6.57,28.56,28.56,0,0,0-5.73,9.6,33.84,33.84,0,0,0,0,22.84,27.87,27.87,0,0,0,15.07,16.22,29.53,29.53,0,0,0,12.36,2.48q9.5,0,16.11-4.3a25.56,25.56,0,0,0,9.65-11.05l-16.28-5.28A9.9,9.9,0,0,1,227.2,371Zm-59-60.82,0,77.32h16.95l0-77.33Zm100.85,15.23V310.08h-17v15.37ZM398,346a27.65,27.65,0,0,0-5.69-9.48,27.15,27.15,0,0,0-9.17-6.53,33.35,33.35,0,0,0-24.87,0,26.78,26.78,0,0,0-9.22,6.58,28.93,28.93,0,0,0-5.73,9.71,34.73,34.73,0,0,0-2,11.78,31.64,31.64,0,0,0,2,11.13,27.83,27.83,0,0,0,5.79,9.37,27.31,27.31,0,0,0,9.28,6.41,31,31,0,0,0,12.36,2.36A29.86,29.86,0,0,0,387,383.07,26.49,26.49,0,0,0,397,372l-14.19-4.16a9.34,9.34,0,0,1-4.53,5.05,14.39,14.39,0,0,1-7,1.86,12.38,12.38,0,0,1-8.19-3c-2.4-2-3.73-5-4-8.83h40.45a20.3,20.3,0,0,0,.31-2.58c.07-1,.11-2,.11-3A33,33,0,0,0,398,346Zm-39.25,6a13.77,13.77,0,0,1,3.7-8.61,10.87,10.87,0,0,1,8.07-3.24,11,11,0,0,1,8,3.18,13.5,13.5,0,0,1,3.75,8.66ZM252.12,331.81v55.61h17l0-55.61Zm66.55-21.72V340.5a19.82,19.82,0,0,0-7.32-7.31,20.63,20.63,0,0,0-10.61-2.71,22.05,22.05,0,0,0-9.82,2.19,24.16,24.16,0,0,0-7.79,6,27.33,27.33,0,0,0-5.09,9,34.12,34.12,0,0,0-1.8,11.18,31.7,31.7,0,0,0,2,11.34,28,28,0,0,0,5.47,9,24.43,24.43,0,0,0,8.33,6,25.69,25.69,0,0,0,10.56,2.14q11,0,16.23-9v7.94h16.87l0-76.29Zm-4.76,61.18a12.42,12.42,0,0,1-6.89,2,13.31,13.31,0,0,1-5.31-1,12.26,12.26,0,0,1-4.19-2.92,13.8,13.8,0,0,1-2.76-4.5,15.56,15.56,0,0,1-1-5.64,17.07,17.07,0,0,1,.9-5.49,13.87,13.87,0,0,1,6.53-7.84,10.3,10.3,0,0,1,5-1.19,12.25,12.25,0,0,1,7.38,2.61,15.35,15.35,0,0,1,5.25,6.89v11.49A13.3,13.3,0,0,1,313.91,371.27Z"/></svg>
```

Comment:
I just picked up the new SVG from the PR for deploying / testing and the SVG logo now shows in black and white and the logo is small too. Basically, the <style>.a{fill:#0031e2;}</style> which fills the logo with blue color is getting removed and rendering in black ‘n white. Please fix this. Thanks